api/jobs.go

```diff
@@ -27,8 +24,8 @@ func (api *JobStoreAPI) CreateJobHandler(ctx context.Context) http.HandlerFunc {
 
 		id := NewID()
 
-		//Create job in job store
-		newJob, err := api.jobStore.CreateJob(ctx, id, sync_mux)
+		//Create job in jobs collection in mongoDB
+		newJob, err := api.mongoDB.CreateJob(ctx, id)
```

Comment:
Again remove references to mongoDB from the API package.
```suggestion
		//Create job in job store
		newJob, err := api.jobStore.CreateJob(ctx, id)
```

Reply:
now changed